application/common/models/Reset.php

```diff
@@ -146,24 +146,22 @@ public function send()
 
     public function sendPrimary()
     {
-        /**
-         * @todo send email to $user->email with reset code
```

Comment:
@fillup, just checking: why did we remove the `@todo` flags before the code was actually written?